### PR TITLE
Homepage starter code

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description

this is the base code for the home page. i created a few components that would come together to make up the home page. 

## Milestones

developing home page

## Resources
my student store project

## Test Plan
Screenshot 2023-07-07 at 2 09 16 PM

### CODE REVIEW

File: timify-api/app.js [hunk 1]
```diff
@@ -10,8 +10,12 @@ app.use(cors())
 app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
 app.use(morgan())
 
-// Route to get all podcasts
 
+app.get('/', (req, res) => {
+  res.send("it works");
```
Comment (ssorenson130) [inline timify-api/app.js:15]: do we need this?

**Review by ssorenson130 — APPROVED** [review]
LGTM! All we need now is some CSS to get everything sized properly inside the grid

File: timify-api/app.js (hunk 1 above)
Comment (PrettyPearl2302) [reply timify-api/app.js:15]: yes i think because i got an error message when i didn't have it.

Comment (ghost) [reply timify-api/app.js:15]: What kind of error message? It seems like unnecessary statement.

Comment (PrettyPearl2302) [reply timify-api/app.js:15]: it said "Cannot GET/ in the console, if i remember correctly

**Pull request merged**